json_serializable/pubspec.yaml

```diff
@@ -8,9 +8,9 @@ environment:
   sdk: '>=2.7.0 <3.0.0'
```

Comment:
Make sure to bump version here, too!

Reply:
Hmm, how should I handle this? I updated CI to run on 2.13.4 so that I could use the newer mono_repo and `dart` CLI, but I can't raise the minimum to that without opting into null safety. Maybe I should use 2.10 instead?

Reply:
We did SDK constraint of `>=2.11.99 <3.0.0` as a flag of "latest that doesn't do null safety"